### DISCUSSION TITLE
Useful ABL snippets

### DISCUSSION THREAD

**Discussion by lievendf** [vscode-abl/vscode-abl#215] [General]
Anyone wants to share useful ABL snippets for VSCode?
Mine: https://gist.github.com/lievendf/290c7b1540adee64363207012351349e

**Comment by cverbiest** [comment]
@lievendf 
We have a large set of snippets, unfortunately a lot of them are custom, using our or SmartComponents tools.

I have quickly split the file in two parts. 
Lowercase version
https://gist.github.com/cverbiest/0b2292c0f0798867a678ed5c606bb375

Not yet fully tested

I'd love to find a way to insert the user/author info, but seems to be missing https://github.com/microsoft/vscode/issues/64599
